sklearn/metrics/_plot/roc_curve.py

```diff
@@ -132,6 +165,11 @@ def plot(self, ax=None, *, name=None, **kwargs):
         ylabel = "True Positive Rate" + info_pos_label
         ax.set(xlabel=xlabel, ylabel=ylabel)
 
+        if plot_chance_level:
+            (self.chance_level_,) = ax.plot((0, 1), (0, 1), **chance_level_line_kw)
```

Comment:
If we reuse ``ax`` for several displays, are we stacking a chance level line that many times ?
Can it have a visual impact ?

Reply:
Understandable. Thanks for the clarification.